pages/collections/[colItemId].js

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+
+import MainLayout from "../../components/MainLayout"
+import ItemView from "../../components/CollectionComponents/ItemView"
```

Comment:
Michael thankfully added aliases for file paths so this can now be `components/CollectionComponents/ItemView` and `components/MainLayout`. I'd say, update the file paths and then you're good to go!

Reply:
:+1: